package.json

```diff
@@ -36,8 +36,11 @@
     "generate:styles": "node scripts/generateStyles.js"
   },
   "dependencies": {
+    "d3-geo": "^3.1.0",
     "dayjs": "^1.11.10",
-    "preact": "^10.12.1"
+    "proj4d3": "^1.0.1",
+    "topojson-client": "^3.1.0",
+    "vite-plugin-svgr": "^4.2.0"
```

Comment:
`vite-plugin-svgr` is not being used is it? Can you please move the other dependencies to `devDependecies` and remove any that are unused?